src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -829,30 +822,63 @@ export default class RadixTransactionBuilder {
         return atom
     }
 
+    private static assertMinNativeTokenBalance(account: RadixAccount, isAtLeast: BN) {
+        const requiredBalance = isAtLeast
```

Comment:
this variable seems unnecessary ? 🤔

Reply:
Tried emulating Swifts amazing function "labels", but sure can remove.